application/helpers/export_helper.php

```diff
@@ -1670,6 +1693,9 @@ function quexml_export($surveyi, $quexmllan, $iResponseID = false)
                     case "D": //DATE
                         $response->appendChild(QueXMLCreateFree("date", "19", ""));
                         quexml_set_default_value($response, $iResponseID, $qid, $iSurveyID, $fieldmap);
+                        if (Yii::app()->getConfig('quexmlkeepsurveydateformat') == true) {
```

Comment:
`=== true` is never needed.

Reply:
Will fix this

Reply:
@olleharstedt you mean `== true` is never needed , `=== true` can be needed ;)

Reply:
@Shnoulle hehe, yeah, with mixed values :)

Reply:
:D